velox/type/HugeInt.h

```diff
@@ -56,6 +56,13 @@ class HugeInt {
   }
 
   static int128_t parse(const std::string& str);
+
+  /// Because value exceeds maxium int64_t cannot write, add this helper
+  /// function to test. It can represent -9223372036854775817223372036854775808
```

Comment:
> Because value exceeds maxium int64_t cannot write, add this helper function to test

Above sentence could be removed and perhaps add more descriptions for the function itself, e.g. what are hi and low, the requirement for them and how the int128_t is created.